river/command/filter.zig

```diff
@@ -36,10 +36,21 @@ pub fn floatFilterAdd(
     if (args.len < 2) return Error.NotEnoughArguments;
     if (args.len > 2) return Error.TooManyArguments;
 
-    const gop = try server.config.float_filter.getOrPut(util.gpa, args[1]);
+    var map = &server.config.float_filter_app_ids;
+    var key = args[1];
+    if (std.mem.startsWith(u8, args[1], "title:")) {
+        key = key[6..];
+        map = &server.config.float_filter_titles;
+    } else {
+        if (std.mem.startsWith(u8, args[1], "app-id:")) {
+            key = key[7..];
+        }
+    }
+
+    const gop = try map.*.getOrPut(util.gpa, key);
```

Comment:
This explicit dereference to call a function in the struct's namespace isn't necessary in zig.
```suggestion
    const gop = try map.getOrPut(util.gpa, key);
```